src/main/kotlin/no/nav/familie/pdf/pdf/TabellUtils.kt

```diff
@@ -11,68 +11,73 @@ import com.itextpdf.layout.properties.UnitValue
 import no.nav.familie.pdf.pdf.domain.VerdilisteElement
 
 object TabellUtils {
-    fun lagTabell(
-        tabellData: List<VerdilisteElement>,
-        caption: String,
+    fun visningsVariantTabeller(
+        verdiliste: List<VerdilisteElement>,
+        seksjon: Div,
+    ) {
+        verdiliste.forEach { barn ->
```

Comment:
```suggestion
        verdiliste.forEach { element ->
```

Fordi denne gjelder typ arbeidsforhold og utenlandsopphold o.l. om jeg skjønner riktig? Om ikke kan du overse kommentaren